From: Hannes Reinecke <hare@suse.de>
Date: Wed, 4 Jul 2012 15:57:05 +0200
Subject: [PATCH] mach-dove: define TIMER_PHYS_BASE
Patch-Mainline: n/a

commit 0fa1f0609a0c1fe8b2be3c0089a2cb48f7fda521 requires the
definition of TIMER_PHYS_BASE, which is missing from mach-dove.
Add it.

Signed-off-by: Hannes Reinecke <hare@suse.de>

diff --git a/arch/arm/mach-dove/include/mach/bridge-regs.h b/arch/arm/mach-dove/include/mach/bridge-regs.h
index 226949d..12ca81e 100644
--- a/arch/arm/mach-dove/include/mach/bridge-regs.h
+++ b/arch/arm/mach-dove/include/mach/bridge-regs.h
@@ -49,6 +49,7 @@
 
 #define POWER_MANAGEMENT	(BRIDGE_VIRT_BASE | 0x011c)
 
+#define TIMER_PHYS_BASE		(BRIDGE_PHYS_BASE | 0x0300)
 #define TIMER_VIRT_BASE		(BRIDGE_VIRT_BASE | 0x0300)
 
 #endif
diff --git a/arch/arm/mach-dove/include/mach/dove.h b/arch/arm/mach-dove/include/mach/dove.h
index ad1165d..5c04e7f 100644
--- a/arch/arm/mach-dove/include/mach/dove.h
+++ b/arch/arm/mach-dove/include/mach/dove.h
@@ -77,6 +77,7 @@
 #define DOVE_SPI1_PHYS_BASE	(DOVE_SB_REGS_PHYS_BASE | 0x14600)
 
 /* North-South Bridge */
+#define BRIDGE_PHYS_BASE	(DOVE_SB_REGS_PHYS_BASE | 0x20000)
 #define BRIDGE_VIRT_BASE	(DOVE_SB_REGS_VIRT_BASE | 0x20000)
 
 /* Cryptographic Engine */
